src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  const onLeftClick = (event) => {
+    if (event.button === 0) {
+      resolve('First promise was resolved');
```

Comment:
To prevent the first promise from being rejected after it has been resolved, you should clear the timeout inside the `onLeftClick` function once the promise is resolved.